kafka-java-liberty-sample/src/main/java/com/eventstreams/samples/servlet/KafkaServlet.java

```diff
@@ -25,11 +115,10 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.StringWriter;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Properties;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalUnit;
+import java.util.*;
```

Comment:
Can we keep the explicit imports?

Reply:
Sure thing - I'll change my IDE settings.